docs/conf.py

```diff
@@ -66,16 +66,16 @@ def __getattr__(cls, name):
 
 # General information about the project.
 project = u'Populus'
-copyright = u'2016, Piper Merriam'
+copyright = u'2018, Piper Merriam'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = "2.0"
+version = "2.2"
 # The full version, including alpha/beta/rc tags.
-release = "2.0.0-alpha.1"
+release = "2.2.0"
```

Comment:
The best option here would be to make the versions automatic. web3.py has an example of detecting the version:

https://github.com/ethereum/web3.py/blob/b058b26/docs/conf.py#L20-L27

and processing those into `version` and `release`: 

https://github.com/ethereum/web3.py/blob/b058b26/docs/conf.py#L65-L67

Are you interested in submitting that change?